src/_posts/databases/postgresql/2000-01-01-changing-plan.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Changing Plan
+nav: Changing Plan
+modified_at: 2024-10-22 00:00:00
+tags: databases postgresql addon
+index: 8
+---
+
+At Scalingo, all databases addons plans are identified using a name made of two
+or three parts separated by a dash (-). This name is made of:
+
+1. a database ***type*** : `postgresql`
+2. a ***class*** : either `sandbox`, `starter` or `business` (learn more about
+   [classes]({% post_url databases/2000-01-01-overview %}#database-plans))
+3. a ***size*** : specifying the amount of RAM available with the plan, in MB
+   (only for starter and business classes).
+
+You can change your PostgreSQL® for Scalingo addon plan whenever you want. The
+operation is launched instantly, no manual input is required.
+
+The impact on your application and the downtime vary depending on several
+factors such as the current plan being used and the one you wish to change for
+(see below for further information).
+
+{% warning %}
+While switching to a more powerful plan is rather safe, you should take extra
+care when changing for a less powerful plan: please make sure the new plan can
+handle all your data and fits your application workload.
+{% endwarning %}
+
+
+## Understanding the Plan Change Process
+
+### From Starter to Starter
+
+When changing the size of a Starter plan, the platform reboots the existing
+instance with the new size. This can lead to a small service interruption
```

Comment:
issue: I think the service interruption is not optional, it will always happen, won't it?
```suggestion
instance with the new size. This leads to a small service interruption
```